### PR TITLE
CCIP 3388 - add offramp generation

### PR DESCRIPTION
**Pull request by ogtownsend**
Add getters and state generation for offramp

### CODE REVIEW

File: integration-tests/deployment/ccip/view/v1_6/feequoter.go [hunk 1]
```diff
@@ -138,6 +138,7 @@ func GetSupportedTokens(taContract *token_admin_registry.TokenAdminRegistry) ([]
 	return tokens, nil
 }
 
+// From the perspective of the OnRamp, the destination chains are the source chains for the OffRamp.
 func GetDestinationSelectors(routerContract *router1_2.Router) ([]uint64, error) {
```
Comment (ogtownsend) [inline integration-tests/deployment/ccip/view/v1_6/feequoter.go]: Thinking it might make more sense to have this defined in `v1_2/router.go`?

Comment (ogtownsend) [reply integration-tests/deployment/ccip/view/v1_6/feequoter.go]: actually maybe not since we're calling it `GetDestinationSelectors`

Comment (AnieeG) [reply integration-tests/deployment/ccip/view/v1_6/feequoter.go]: How about `GetRemoteChainSelectors` ?

File: integration-tests/deployment/ccip/view/v1_6/offramp.go [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+		}
+		sourceChainConfigs[sourceChainSelector] = sourceChainConfig
+	}
+
+	staticConfig, err := offRampContract.GetStaticConfig(nil)
+	if err != nil {
+		return OffRampView{}, fmt.Errorf("failed to get static config: %w", err)
+	}
+
+	owner, err := offRampContract.Owner(nil)
```
Comment (AnieeG) [inline integration-tests/deployment/ccip/view/v1_6/offramp.go]: this is part of [NewContractMetaData](https://github.com/smartcontractkit/chainlink/blob/develop/integration-tests/deployment/ccip/view/types/contract_state.go#L19)

File: integration-tests/deployment/ccip/view/v1_6/offramp.go [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+	router1_2 "github.com/smartcontractkit/chainlink/v2/core/gethwrappers/ccip/generated/router"
+)
+
+type OffRampView struct {
+	types.ContractMetaData
+	DynamicConfig             offramp.OffRampDynamicConfig                `json:"dynamicConfig"`
+	LatestPriceSequenceNumber uint64                                      `json:"latestPriceSequenceNumber"`
+	SourceChainConfigs        map[uint64]offramp.OffRampSourceChainConfig `json:"sourceChainConfigs"`
+	StaticConfig              offramp.OffRampStaticConfig                 `json:"staticConfig"`
+	Owner                     common.Address                              `json:"owner"`
```
Comment (AnieeG) [inline integration-tests/deployment/ccip/view/v1_6/offramp.go]: This is part of ContractMetaData

File: integration-tests/deployment/ccip/view/v1_6/offramp.go (hunk 2 above)
Comment (ogtownsend) [reply integration-tests/deployment/ccip/view/v1_6/offramp.go]: updated

File: integration-tests/deployment/ccip/view/v1_6/offramp.go (hunk 3 above)
Comment (ogtownsend) [reply integration-tests/deployment/ccip/view/v1_6/offramp.go]: updated

**Review by AnieeG — APPROVED** [review]

**Review by cmalec — APPROVED** [review]

**Pull request merged**